src/utils.ts

```diff
@@ -82,6 +82,8 @@ export class Utils {
             return jfrogCredentials;
         }
 
+        // Used for usage reporting
+        this.exportVariableIfNotSet("JFROG_CLI_USAGE_CONFIG_OIDC","TRUE")
```

Comment:
Please ensure to run `npm run format` before the commit
```suggestion
        this.exportVariableIfNotSet("JFROG_CLI_USAGE_CONFIG_OIDC", "TRUE")
```